readthedocs/builds/filters.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from django.forms.widgets import HiddenInput
+from django.utils.translation import ugettext_lazy as _
+from django_filters import CharFilter, ChoiceFilter, FilterSet
+
+from readthedocs.builds.constants import BUILD_STATE_FINISHED
+
+log = logging.getLogger(__name__)
+
+
+class BuildListFilter(FilterSet):
+
+    STATE_ACTIVE = 'active'
+    STATE_SUCCESS = 'succeeded'
+    STATE_FAILED = 'failed'
+
+    STATE_CHOICES = (
+        (STATE_ACTIVE, _('Active')),
+        (STATE_SUCCESS, _('Build finished')),
+        (STATE_FAILED, _('Build failed')),
+    )
```

Comment:
Should these live in a constants file somewhere? Seems useful outside of just the filter.

Reply:
These are very particular to the filter UI, and I wouldn't expect to reuse them.